.gitmodules

```diff
@@ -132,7 +132,11 @@
 	path = math-libs/support/mxDataGenerator
 	url = https://github.com/ROCm/mxDataGenerator.git
 	branch = main
+[submodule "composable_kernel"]
+	path = ml-libs/composable_kernel
+	url = https://github.com/ROCm/composable_kernel.git
+	branch = develop
 [submodule "rocRoller"]
 	path = math-libs/BLAS/rocRoller
 	url = https://github.com/ROCm/rocRoller.git
-	branch = main
+	branch = main
```

Comment:
pre-commit has a few complaints about newlines across this PR: https://github.com/ROCm/TheRock/actions/runs/15766388687/job/44443586156?pr=876

Easiest way to fix is to install https://pre-commit.com/ and run `pre-commit run --all-files`